packages/sdk/src/global-account/react/components/ManageAccount/TokenContent.tsx

```diff
@@ -0,0 +1,72 @@
+import { useSimBalance } from "@b3dotfun/sdk/global-account/react";
+import { useActiveWallet } from "thirdweb/react";
+import { TokenBalanceRow } from "./TokenBalanceRow";
+
+// Chain logo URLs mapping
+const CHAIN_LOGOS: Record<number, string> = {
+  1: "https://ethereum-optimism.github.io/data/ETH/logo.svg", // Ethereum
+  8453: "https://avatars.githubusercontent.com/u/108554348?s=280&v=4", // Base
+  8333: "https://cdn.b3.fun/b3-logo-white-circle.svg", // B3
+};
+
+// Native token logos (ETH across chains)
+const ETH_LOGO = "https://ethereum-optimism.github.io/data/ETH/logo.svg";
```

Comment:
I believe we show this elsewhere too? Can we unify the logos for chains overall in constants dir? We have `packages/sdk/src/shared/utils/chains.ts`, `apps/global-accounts/src/constants/b3Chain.tsx`, `packages/sdk/src/shared/constants/chains/supported.ts`, etc. and i believe might hard-code elsewhere too. In some places, like AnySpend, maybe we're pulling tokens even from API? Just wanna make sure it's unified